src/features/settings/ui/SettingsModal.module.css

```
@@ -0,0 +1,162 @@
+  z-index: 1000;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 24px;
+  bottom: 130px;
+}
+
+.modal {
+  --panel-horizontal-padding: 16px;
+  --panel-content-width: 1060px;
+  position: relative;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)